docs/sources/upgrade-guide/when-to-upgrade/index.md

```diff
@@ -18,6 +18,10 @@ We no longer wait for the yearly major release to give you access to the next bi
 
 Having a dependable release process provides users like you with the best Grafana experience possible, and it provides the flexibility to upgrade in a manner that works best for you and your organization.
 
+{{< admonition type="note" >}}
+Grafana Cloud follows a different release cadence than Grafana OSS and Enterprise. To learn more about Cloud release channels, refer to [Rolling release channels for Grafana Cloud](https://grafana.com/docs/rolling-release/).
+{{< /admonition >}}
```

Comment:
Tricky to word here, but is it okay that we are generalizing "Grafana Cloud", the whole suite is not on RRC, just Grafana Core

```suggestion
{{< admonition type="note" >}}
Grafana Cloud follows a different release cadence than Grafana OSS and Enterprise. Grafana Cloud, much like many other SaaS product, has different deployment cadences for various parts of the application suite. Grafana Core in Grafana Cloud uses Rolling release channels. To learn more about release channels, refer to [Rolling release channels for Grafana Cloud](https://grafana.com/docs/rolling-release/).
{{< /admonition >}}
```